packages/gui/src/components/settings/SettingsIntegration.tsx

```diff
@@ -45,6 +46,9 @@ export default function SettingsIntegration() {
     useWalletConnectPairs();
 
   const pairs = get();
+  const { data: fingerprint } = useGetLoggedInFingerprintQuery();
+
+  const [bypassReadonlyCommands, setBypassReadonlyCommands] = useLocalStorage<any>('bypass-readonly-commands', {});
```

Comment:
Is there a reason not to use the `useWalletConnectPairs` or `useWalletConnectPreferences` hooks? Creating a new localStorage item is going to make maintenance more complex. At the very least we should make the naming of these localStorage items consistent -- maybe `walletConnectBypassReadonlyCommands`. But I would prefer using the existing hooks if possible.